presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -263,6 +274,20 @@ public void testNoCacheExceptions()
         assertEquals(mockClient.getAccessCount(), 2);
     }
 
+    @Test
+    public void testServiceCreationWithOnlyTTLEnabled()
+    {
+        CachingHiveMetastoreConfig config = new CachingHiveMetastoreConfig();
+        config.setMetastoreCacheTtl(new Duration(10, TimeUnit.MILLISECONDS));
+
+        CachingHiveMetastore metastore = new CachingHiveMetastore(
+                new BridgingHiveMetastore(createThriftHiveMetastore()),
+                createTestThreadPool(),
```

Comment:
Please use direct executor here.

Reply:
I would wrap it all in separate methods to increase the fluency of reading these tests, but I know how much you don't like that ;) 
`CachingHiveMetastore createMetastore(...)`